src/app/accounts/register.component.ts

```diff
@@ -65,5 +68,25 @@ export class RegisterComponent extends BaseRegisterComponent {
                 }
             } catch { }
         }
+
+        if (this.policies != null) {
+            this.enforcedPolicyOptions = await this.policyService.getMasterPasswordPolicyOptions(this.policies);
+        }
+    }
+
+    async submit() {
+        if (this.enforcedPolicyOptions == null) {
+            super.submit();
+            return;
+        }
```

Comment:
This seems unnecessary. Can we just add the `this.enforcedPolicyOptions == null` check in the other if statement and remove all of this?